lacework/resource_lacework_vulnerability_exception_host.go

```diff
@@ -317,6 +335,14 @@ func resourceLaceworkVulnerabilityExceptionHostUpdate(d *schema.ResourceData, me
 		}
 	)
 
+	if d.Get("expiry") != nil {
+		expiryTime, err = time.Parse(time.RFC3339, d.Get("expiry").(string))
+		if err != nil {
+			return errors.Wrap(err, fmt.Sprintf("unable to parse expiryTime %s", d.Get("expiry").(string)))
```

Comment:
where is `expiryTime` defined? 🤔 maybe this is better?
```suggestion
			return errors.Wrap(err, fmt.Sprintf("unable to parse expiry time %s", d.Get("expiry").(string)))
```